### PR TITLE
269828/setup alert component using msds components

### PR DESCRIPTION
**Pull request by mattsskea**

### CODE REVIEW

File: components/alert/alert.hbs [hunk 1]
```diff
@@ -22,7 +22,7 @@
 							Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas varius tortor nibh, sit amet tempor nibh finibus et. Aenean eu enim justo. Vestibulum aliquam hendrerit molestie. Mauris malesuada nisi sit amet augue accumsan tincidunt. Maecenas tincidunt, velit ac porttitor pulvinar.
 						</p>
 						<div class="msds-alert__action-container">
-							 <button type="button" class="msds-btn msds-btn--secondary msds-btn--sm">ok, i get it</button>
+							 <button type="button" class="msds-btn msds-btn--secondary msds-btn--sm js-msds-alert-close-click-handler">ok, i get it</button>
```
Comment (ms-Danny) [inline components/alert/alert.hbs:25]: why create a seperate class for an eventhandler?

Comment (mattsskea) [reply components/alert/alert.hbs:25]: This was implemented by Martin way back when, I am utilizing what already exists

File: components/alert/alert.hbs [hunk 2]
```diff
@@ -1,32 +1,102 @@
...
-							 <button type="button" class="msds-btn msds-btn--secondary msds-btn--sm">ok, i get it</button>
- 
+							<button type="button"
+								class="msds-btn msds-btn--secondary msds-btn--sm">ok,
+								i get it</button>
+
 							<div class="msds-alert__checkbox-container">
-								<input type="checkbox" class="msds-alert__checkbox" id="msds-alert__checkbox" type="text">
-								<label class="msds-alert__checkbox-label" for="msds-alert__checkbox">Don't show this again</label>
+								<input type="checkbox" class="msds-alert__checkbox" id="msds-alert__checkbox"
```
Comment (ms-Danny) [inline components/alert/alert.hbs:96]: Would it make sense to reuse the component here in the checkbox component for the forms and then import it here aswell?

Comment (mattsskea) [reply components/alert/alert.hbs:96]: we don't have the checkbox component yet to my knowledge but yes I agree that would be the right way to go

**Pull request merged**